components/user-popover.js

```diff
@@ -0,0 +1,79 @@
+import { USER_FULL } from '@/fragments/users'
+import { NORMAL_POLL_INTERVAL, SSR } from '@/lib/constants'
+import Moon from '@/svgs/moon-fill.svg'
+import { useLazyQuery } from '@apollo/client'
+import Link from 'next/link'
+import { useRef, useState } from 'react'
+import { Popover } from 'react-bootstrap'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import { User } from './user-list'
+import styles from './user-popover.module.css'
+import errorStyles from '@/styles/error.module.css'
+
+function StackingSince ({ since }) {
+  return (
+    <small className='text-muted d-flex-inline'>stacking since: {since
+      ? <Link href={`/items/${since}`}>#{since}</Link>
+      : <span>never</span>}
+    </small>
+  )
+}
+
+export default function UserPopover ({ name, children }) {
+  const [showOverlay, setShowOverlay] = useState(false)
+
+  const [getUser, query] = useLazyQuery(
+    USER_FULL,
+    SSR
```

Comment:
with `useLazyQuery` you don't need this check nor should you poll.

Reply:
Cool!

Fixed: 590c57109432d300af3265297e5ea2242a34fe4a